tools/install.sh

```diff
@@ -160,19 +160,19 @@ os_distro=$(hostnamectl |awk -F: '/Operating System:/{print $2}'|cut -f2 -d ' ')
 debian=("Debian" "Ubuntu")
 redhat=("Red" "CentOS" "Fedora" "Oracle") 
 
-if [[ " ${debian[*]} " =~ " $os_distro " ]]; then
-    mkdir -p ../modules/document_repository/user_uploads
```

Comment:
Why are these `-p` flags removed?

Reply:
Some mistake during rebase probably.
Will add them back

Reply:
Where probably removed by the rebase. 
They are not needed as the individual modules repositories already exist and it just create 1 level of extra directories.

Reply:
They are needed because `user_uploads/` is not tracked by git. #5245 #5050 were issued to add `-p`.

Please add them back.

Reply:
'-p' is required when you want to create multiple level of subdir in one instruction. In this case, both '../modules/document_repository/' and '../modules/data_release/' already exist as they are managed by Git. Thus, only one level of subdir need to be create 'user_uploads', '-p' is not required